src/Pororoca.Desktop/ViewModels/WebSocketConnectionViewModel.cs

```diff
@@ -681,7 +688,11 @@ private async Task CollectExchangedMessagesAsync(ChannelReader<WebSocketMessage>
     {
         await foreach (var msg in channelReader.ReadAllAsync())
         {
-            ExchangedMessages.Insert(0, new(msg, DateTimeOffset.Now));
+            //ExchangedMessages.Insert(0, new(msg, DateTimeOffset.Now));
+            var vm = new WebSocketExchangedMessageViewModel(msg, DateTimeOffset.Now);
+            ExchangedMessages.Insert(0, vm);
+            if (vm.CanBeSavedToFile)
```

Comment:
If msg 1 can be saved to file and the following msg 2 can't, then the button will disappear; this if-clause should be something like:

```cs
if (ExchangedMessages.Any(msg => msg.CanBeSavedToFile))
{
    IsSaveAllExchangedMessagesToFilesVisible = true;
}
```